### PR TITLE
Add lightning mutations to wallet id middleware

### PR DESCRIPTION
**Pull request by dolcalmi**
Add lnInvoiceCreate, lnNoAmountInvoiceCreate, lnInvoicePaymentSend, lnNoAmountInvoicePaymentSend to wallet id middleware

### CODE REVIEW

**Review by samerbuna — APPROVED** [review]

**Review by nicolasburtey — COMMENTED** [review]
what does this PR enables?

**Comment by dolcalmi** [comment]
> what does this PR enables?

this is the last PR of walletId middleware/graphql implementation (for V2) please check https://github.com/GaloyMoney/galoy/pull/741, https://github.com/GaloyMoney/galoy/pull/743 and https://github.com/GaloyMoney/galoy/pull/744

File: src/app/wallets/add-invoice-for-wallet.ts [hunk 1]
```diff
@@ -16,6 +16,22 @@ import { LndService } from "@services/lnd"
 import { WalletInvoicesRepository, WalletsRepository } from "@services/mongoose"
 import { RedisRateLimitService } from "@services/rate-limit"
 
+export const addInvoiceByWalletPublicId = async ({
```
Comment (nicolasburtey) [inline src/app/wallets/add-invoice-for-wallet.ts:19]: currently invoices are still be created by `user.id`?

Comment (vindard) [reply src/app/wallets/add-invoice-for-wallet.ts:19]: Yup current code is this ([here](https://github.com/GaloyMoney/galoy/pull/746/files/5a79ed3824b0546e7b23311716a5e889a0fac87b#diff-db3c5b3e8b3f24796d542d4e5c92a62bf101618085c4776bd53c2e0124d4cfc4L32))

**Review by vindard — APPROVED** [review]

**Review by bodymindarts — APPROVED** [review]

**Pull request merged**